storybook/storybook-react/src/Alert/Alert.stories.tsx

```diff
@@ -12,28 +12,21 @@ const meta = {
   args: {
     title: 'Let op',
     closeable: false,
-    icon: false,
   },
   argTypes: {
     severity: {
       control: {
         type: 'radio',
-        labels: { undefined: 'default', error: 'error', success: 'success' },
+        labels: { warning: 'warning', success: 'success', error: 'error', info: 'info' },
```

Comment:
I don't think the labels are necessary if they're the same as the options

Reply:
Correct